### ISSUE TITLE
Material theme, DOI overflows container on change of text spacing

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4265]
### Describe the inaccessible feature
WCAG 1.4.12 Text spacing failure - see #4260 

DOI overflows the container at the required spacing.  Other containers wrap.

Default:
<img width="484" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/5dd028f3-e3a3-4b16-b610-e2db1a49a4c7">

With custom spacing:
<img width="527" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/2a9e5126-2a9f-4f9c-8c20-a3c784ab59a8">
